zds/tutorialv2/utils.py

```diff
@@ -962,3 +962,11 @@ class BadArchiveError(Exception):
 
     def __init__(self, reason):
         self.message = reason
+
+
+class FailureDuringPublication(Exception):
+    """Exception raised if something goes wrong during publication process
```

Comment:
J’ai peut-être tord mais je mettrai un “the” dans ce cas, vu que c’est au singulier et qu’on ne parle pas d’une grandeur indénombrable : 

> Exception raised if something goes wrong during the publication process